spec/omniauth/strategies/signnow_spec.rb

```diff
@@ -103,15 +103,13 @@
     let(:base64_client_key) {
       Base64.strict_encode64(subject.client.id + ':' + subject.client.secret)
     }
-    it  'is protected' do
-      expect(subject.auth_options).to raise_error(NoMethodError)
-    end
+
     it 'returns a hash with the Autorization header' do
-      expect(subject.send(:auth_options).class).to eql(Hash)
-      expect(subject.send(:auth_options)).to include(:headers)
-      expect(subject.send(:auth_options)[:headers]).to include :Authorization
-      expect(subject.send(:auth_options)[:headers][:Authorization]).to match(/Basic/)
-      expect(subject.send(:auth_options)[:headers][:Authorization]).to match(/#{base64_client_key}/)
+      expect(subject.auth_options.class).to eql(Hash)
```

Comment:
This spec was failing, I removed the check for protected since it was not (checked other omniauth gems and seems like this is OK like that)